config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'password_resets/new'
+  get 'password_resets/edit'
```

Comment:
Also, you can remove these lines since you used `resources :password_resets, only: [:new, :create, :edit, :update]`.